SwiftTweaks/TweakTableCell.swift

```diff
@@ -310,8 +369,37 @@ extension TweakTableCell: UITextFieldDelegate {
 			} else {
 				updateSubviews()
 			}
-		case .boolean:
+		case .boolean, .optionsList:
 			assertionFailure("Shouldn't be able to update text field with a Boolean tweak.")
 		}
 	}
 }
+
+
+
+extension TweakTableCell: UIPickerViewDelegate, UIPickerViewDataSource {
+    
+    func numberOfComponents(in pickerView: UIPickerView) -> Int {
+        return 1
```

Comment:
This might serve our purpose but if we want to contribute back to the library, we have to make this configurable